packages/lesswrong/server/migrations/20221222T014643.auto.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Generated on 2022-12-22T01:46:43.544Z by `yarn makemigrations`
+ * The following schema changes were detected:
+ * -------------------------------------------
+ * ***Diff too large to display***
+ * -------------------------------------------
+ * (run `git diff --no-index schema/accepted_schema.sql schema/schema_to_accept.sql` to see this more clearly)
+ *
+ * - [ ] Write a migration to represent these changes
+ * - [ ] Rename this file to something more readable if you wish
+ * - [ ] Uncomment `acceptsSchemaHash` below
+ * - [ ] Run `yarn acceptmigrations` to update the accepted schema hash (running makemigrations again will also do this)
+ */
+export const acceptsSchemaHash = "b06a4e185cab354eccd7e4b1255cfa7a";
+
+export const up = async ({db}: MigrationContext) => {
+  // TODO
```

Comment:
Is one of the generated migration files redundant?

Reply:
I think so? I kind of want to just ignore the whole postgres-schema-migrations thing until it's in a state where it can handle simple field additions without manual intervention but I guess I'm forced to deal with it.